aws/table_aws_accessanalyzer_finding.go

```diff
@@ -0,0 +1,281 @@
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("AccessAnalyzerArn"),
+			},
+			{
+				Name:        "id",
+				Description: "The ID of the finding.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.Id"),
+			},
+			{
+				Name:        "action",
+				Description: "The action in the analyzed policy statement that an external principal has permission to use.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Finding.Action"),
+			},
+			{
+				Name:        "analyzed_at",
+				Description: "The time at which the resource-based policy that generated the finding was analyzed.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("Finding.AnalyzedAt"),
+			},
+			{
+				Name:        "condition",
+				Description: "The condition in the analyzed policy statement that resulted in a finding.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Finding.Condition"),
+			},
+			{
+				Name:        "created_at",
+				Description: "The time at which the finding was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("Finding.CreatedAt"),
+			},
+			{
+				Name:        "error",
+				Description: "The error that resulted in an Error finding.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.Error"),
+			},
+			{
+				Name:        "is_public",
+				Description: "Indicates whether the finding reports a resource that has a policy that allows public access.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromField("Finding.IsPublic"),
+			},
+			{
+				Name:        "principal",
+				Description: "The external principal that has access to a resource within the zone of trust.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Finding.Principal"),
+			},
+			{
+				Name:        "resource",
+				Description: "The resource that the external principal has access to.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.Resource"),
+			},
+			{
+				Name:        "resource_owner_account",
+				Description: "The Amazon Web Services account ID that owns the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.ResourceOwnerAccount"),
+			},
+			{
+				Name:        "resource_type",
+				Description: "The type of the resource that the external principal has access to.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Finding.ResourceType"),
+			},
+			{
+				Name:        "sources",
+				Description: "The sources of the finding, indicating how the access that generated the finding is granted. It is populated for Amazon S3 bucket findings.",
+				Type:        proto.ColumnType_JSON,
```

Comment:
Could you please rearrange the columns? All the JSON columns should be placed towards the down.